R/options.R

```diff
@@ -90,6 +90,9 @@ pt_options <- function(...) {
 #' @param additional_scripts
 #' A character vector containing file paths to any additional scripts which should be included. These will be sourced with includeScript.
 #' @param logo_position Which side should the logo be on? Can be "left" or "right".
+#' @param on_start_fun An optional function to execute when the Shiny server function begins.
```

Comment:
For these other two new optional functions, can we also request `...` as parameters? That means we retain back-compatibility if we add new arguments later.